### PR TITLE
Adds icons and text strings for image build status

### LINKED ISSUE #34: Create Status component for Images table
Depending on the [status of the image](https://github.com/osbuild/osbuild-composer/blob/master/internal/cloudapi/openapi.yml#L84), specific text strings and icons should display in the table that lists images, as shown in the [design doc](https://docs.google.com/presentation/d/1YUPsj01oafTfnTqlihf6-stGbuINfQSIN9K9a24-byY/edit?usp=sharing).

### PR DESCRIPTION
**Pull request by jgiardino**
This update provides a status component that replaces the [raw status value](https://github.com/osbuild/osbuild-composer/blob/master/internal/cloudapi/openapi.yml#L84) with an icon and text string depending on the status value provided. The icon and text are based on [examples in the design](https://docs.google.com/presentation/d/1YUPsj01oafTfnTqlihf6-stGbuINfQSIN9K9a24-byY/edit#slide=id.g9551495f59_6_0). An example of each status state is shown in the example below. 

Some notes:
- One difference between this implementation and the design is that the design shows a link to get an error code for the failed status, but this link should be added after support for this exists.
- This status component is specific to images that are uploaded, which is what we initially plan to support. When support for downloading images is provided, it should be updated to provide the status we would show for downloadable images.

Closes #34 

image

### CODE REVIEW

**Comment by jgiardino** [comment]
@Gundersanne @jkozol - I haven't added anything to tests for this component. Do you have any suggestions on what should be tested? And would this test be in the context of LandingPage.test.js?

@katierik @andreasn - Will you review the example I posted and let me know if the various messages in the Status column match what you were expecting?

**Comment by Gundersanne** [comment]
@jgiardino I pushed a commit to your branch, adding a test and rebasing it to include the extra tests.

**Comment by katierik** [comment]
@jgiardino In the screenshot its hard to tell - is the processing indicator animated?

**Comment by Gundersanne** [comment]
@katierik yup, it's a spinner thingy

**Comment by katierik** [comment]
Then I'm 👍

**Comment by andreasn** [comment]
Looks good to me!

File: src/test/SmartComponents/LandingPage/ImagesTable.test.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```
Comment (atodorov) [inline src/test/SmartComponents/LandingPage/ImagesTable.test.js:1]: File name should be SmartComponents/ImagesTable/ImagesTable.test.js IMO.

**Review by atodorov — CHANGES_REQUESTED** [review]
Other than the location of the test file the tests look good to me.

**Review by atodorov — APPROVED** [review]
Test LGTM, feel free to merge.

**Pull request merged**